packages/flutter/lib/src/widgets/scroll_metrics.dart

```diff
@@ -36,7 +36,7 @@ abstract class ScrollMetrics {
   ///
   /// This is useful if this object is mutable, but you want to get a snapshot
   /// of the current state.
-  ScrollMetrics cloneMetrics() => new FixedScrollMetrics.clone(this);
+  ScrollMetrics toMetrics() => new FixedScrollMetrics.clone(this);
```

Comment:
This change doesn't make sense because it's already metrics.

Also this would be a breaking API change and we've entered the API stability phase of development, where we're trying really hard to not make changes to APIs unless we feel they are very strongly justified.